docs/essentials/getting-started.md

```diff
@@ -17,6 +17,8 @@ Once you have the plugin imported and Unity has had a chance to recompile succes
 
 ![](./getting-started/package-manager.png "An empty project with Normcore freshly imported.")
 
+The plugin will have also added the [Normal scoped registry](#normal-scoped-registry) to the project.
```

Comment:
This wording is a little awkward. Maybe "The plugin will automatically add the [Normal scoped registry] to the project." ?